packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -174,82 +126,76 @@ export function usePriority() {
+): MappedVisitQueueEntry => ({
+  id: queueEntry.uuid,
+  encounters: queueEntry.visit?.encounters?.map(mapEncounterProperties),
+  name: queueEntry.display,
+  patientUuid: queueEntry.patient.uuid,
+  patientAge: queueEntry.patient.person?.age + '',
+  patientSex: queueEntry.patient.person?.gender === 'M' ? 'MALE' : 'FEMALE',
+  patientDob: queueEntry?.patient?.person?.birthdate
+    ? formatDate(parseDate(queueEntry.patient.person.birthdate), { time: false })
+    : '--',
+  // Map `Urgent` to `Priority` because it's easier to distinguish between tags named
+  // `Priority` and `Not Urgent` rather than `Urgent` vs `Not Urgent`
+  priority: queueEntry.priority.display === 'Urgent' ? 'Priority' : queueEntry.priority.display,
+  priorityComment: queueEntry.priorityComment,
+  priorityUuid: queueEntry.priority.uuid,
+  service: queueEntry.queue.name,
+  status: queueEntry.status.display as QueueStatus,
+  statusUuid: queueEntry.status.uuid,
+  waitTime: queueEntry.startedAt ? `${dayjs().diff(dayjs(queueEntry.startedAt), 'minutes')}` : '--',
+  visitStartDateTime: queueEntry.startedAt,
+  visitType: queueEntry.visit?.visitType?.display,
+  visitLocation: queueEntry.visit?.location?.uuid,
+  queueLocation: (queueEntry?.queue as any)?.location?.uuid,
+  visitTypeUuid: queueEntry.visit?.visitType?.uuid,
+  visitUuid: queueEntry.visit?.uuid,
+  queueUuid: queueEntry.queue.uuid,
+  queueEntryUuid: queueEntry.uuid,
+  sortWeight: queueEntry.sortWeight,
+  visitQueueNumber: queueEntry.visit?.attributes?.find((e) => e.attributeType.uuid === visitQueueNumberAttributeUuid)
+    ?.value,
+  identifiers: queueEntry.patient?.identifiers as Identifer[],
+  queueComingFrom: queueEntry?.queueComingFrom?.name,
+});
+
 export function useVisitQueueEntries(currServiceName: string, locationUuid: string): UseVisitQueueEntries {
   const { queueLocations } = useQueueLocations();
   const queueLocationUuid = locationUuid ? locationUuid : queueLocations[0]?.id;
-  const config = useConfig();
-  const { visitQueueNumberAttributeUuid } = config;
 
-  const apiUrl = `${restBaseUrl}/visit-queue-entry?location=${queueLocationUuid}&v=full`;
+  const apiUrl =
+    `${restBaseUrl}/visit-queue-entry?v=full` + (queueLocationUuid ? `&location=${queueLocationUuid}` : '');
   const { t } = useTranslation();
   const { data, error, isLoading, isValidating, mutate } = useSWR<{ data: { results: Array<VisitQueueEntry> } }, Error>(
     apiUrl,
     openmrsFetch,
   );
 
-  const mapEncounterProperties = (encounter: Encounter): MappedEncounter => ({
-    diagnoses: encounter.diagnoses,
-    encounterDatetime: encounter.encounterDatetime,
-    encounterType: encounter.encounterType.display,
-    obs: encounter.obs,
-    provider: encounter.encounterProviders[0]?.provider?.person?.display,
-    uuid: encounter.uuid,
-    voided: encounter.voided,
-  });
-
-  const mapVisitQueueEntryProperties = (visitQueueEntry: VisitQueueEntry): MappedVisitQueueEntry => ({
```

Comment:
refactored `mapVisitQueueEntryProperties` so it is a global function useable elsewhere.